### PR TITLE
feat: #2349 Fix POST response location header to be relative path to …

### PR DESCRIPTION
**Pull request by markkopaas**
…list endpoint, meaning id.

### CODE REVIEW

File: src/data-source.handlers.ts [hunk 1]
```diff
@@ -24,7 +23,7 @@ export const dataSourceHandlers: ResourceHandlerMap = {
       .save()
       .then(datasource => {
         res
-          .location(`${dataSourcesPath}/${datasource.id}`)
+          .location(datasource.id)
```
Comment (azaxarov) [inline src/data-source.handlers.ts]: Although the URI is not exactly meaningful or unique that way, it's easier to parse on the FE

Comment (Chlenix) [reply src/data-source.handlers.ts]: Oh actually I think it's necessary with a / in the beginning if it's relative. The idea is to be able to do originalUrl + location. Original url here being `{host}/api/datasources` and so location should be `'/${datasource.id}'` and not just `datasource.id`

Comment (markkopaas) [reply src/data-source.handlers.ts]: jeg tror det ikke. / mener absolutt fra host.

Comment (Chlenix) [reply src/data-source.handlers.ts]: ok!

**Pull request merged**